README.md

```diff
@@ -246,6 +275,7 @@ it.
 
 Unlike `corepack use` this command doesn't take a package manager name nor a
 version range, as it will always select the latest available version from the
+range specified in `devEngines.packageManager.version`, or fallback to the
```

Comment:
What about top-level `packageManager`?

Reply:
That's what `or fallback to the same major line` is for